### ISSUE TITLE
PowerToys FancyZones freezes launching with VSCode on Windows

### ISSUE THREAD

**Issue by TortillaZHawaii** [flutter/flutter#107537]
## Steps to Reproduce

1. Create an app using `flutter create`.
2. Use vscode to launch the app.
3. Move launched app using [PowerToys FancyZones](https://docs.microsoft.com/en-us/windows/powertoys/fancyzones).
4. Stop the app.
5. Launch the app again using vscode.

Recorded steps: https://youtu.be/jCw_g4I3zVE

**Expected results:** the app launches without issues.

**Actual results:** the app freezes with white rectangle within default app screen position, and translucent background in the rest of remaining window: 
![image](https://user-images.githubusercontent.com/62249621/178741708-37eb58eb-5f6b-45b4-9405-416275237e96.png)

<details>
<summary>Code sample</summary>
App created using `flutter create`.

</details>

<details>
  <summary>Logs</summary>

Logged using vscode with `launch.json`:

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch",
            "type": "dart",
            "request": "launch",
            "args": ["--verbose"],
        }
    ]
}
```

after last log the app freezes.

```
[  +77 ms] executing: [C:\src\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +96 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] 85684f9300908116a78138ea4c6036c35c9a1236
[        ] executing: [C:\src\flutter/] git tag --points-at 85684f9300908116a78138ea4c6036c35c9a1236
[  +87 ms] Exit code 0 from: git tag --points-at 85684f9300908116a78138ea4c6036c35c9a1236
[        ] 3.0.4
[   +7 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +64 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [C:\src\flutter/] git ls-remote --get-url origin
[  +59 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +134 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref HEAD
[  +63 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[ +100 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +4 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +73 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +6 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +91 ms] Skipping pub get: version match.
[ +135 ms] Generating D:\bugs\windows_example\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +127 ms] Initializing file store
[  +11 ms] Skipping target: gen_localizations
[   +5 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents: D:\bugs\windows_example\.dart_tool\package_config_subset}
[  +35 ms] gen_dart_plugin_registrant: Complete
[   +1 ms] Skipping target: _composite
[   +1 ms] complete
[   +6 ms] Launching lib\main.dart on Windows in debug mode...
[   +5 ms] C:\src\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\src\flutter\bin\cache\dart-sdk\bin\snapshots\frontend_server.dart.snapshot --sdk-root C:\src\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names --experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill C:\Users\dwyso\AppData\Local\Temp\flutter_tools.517004\flutter_tool.a7d1d369\app.dill --packages D:\bugs\windows_example\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill build\c075001b96339384a97db4862b8ab8db.cache.dill.track.dill --enable-experiment=alternative-invalidation-strategy
[  +26 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[ +166 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[        ] []
[        ] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.VCTools Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[ +168 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.VCTools Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[        ] [
             {
               "instanceId": "74e57e92",
               "installDate": "2021-08-03T18:17:15Z",
               "installationName": "VisualStudio/16.11.16+32602.291",
               "installationPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools",
               "installationVersion": "16.11.32602.291",
               "productId": "Microsoft.VisualStudio.Product.BuildTools",
               "productPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\Common7\\Tools\\LaunchDevCmd.bat",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Build Tools 2019",
               "description": "The Visual Studio Build Tools allows you to build native and managed MSBuild-based applications without requiring the Visual Studio IDE. There are options to install the Visual C++ compilers and libraries, MFC, ATL, and C++/CLI support.",
               "channelId": "VisualStudio.16.Release",
               "channelUri": "https://aka.ms/vs/16/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "installedChannelUri": "https://aka.ms/vs/16/release/channel",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-v16.11#16.11.16",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=660909",
               "updateDate": "2022-06-21T15:43:04.2772238Z",
               "catalog": {
                 "buildBranch": "d16.11",
                 "buildVersion": "16.11.32602.291",
                 "id": "VisualStudio/16.11.16+32602.291",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "16.11.16",
                 "productLine": "Dev16",
                 "productLineVersion": "2019",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "16",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "16.11.16+32602.291",
                 "requiredEngineVersion": "2.11.69.53063"
               },
               "properties": {
                 "campaignId": "",
                 "channelManifestId": "VisualStudio.16.Release/16.11.16+32602.291",
                 "nickname": "2",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[  +31 ms] <- compile package:windows_example/main.dart
[   +2 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S D:\bugs\windows_example\windows -B build\windows -G Visual Studio 16 2019
[ +127 ms] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.22000.
[  +41 ms] -- Configuring done
[ +156 ms] -- Generating done
[   +7 ms] -- Build files have been written to: D:/bugs/windows_example/build/windows
[  +12 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Debug --target INSTALL --verbose
[ +231 ms] Microsoft (R) Build Engine version 16.11.2+f32259642 for .NET Framework
[        ] Copyright (C) Microsoft Corporation. All rights reserved.
[  +64 ms] Build started 13/07/2022 15:20:58.
[ +297 ms] Project "D:\bugs\windows_example\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[        ] Project "D:\bugs\windows_example\build\windows\INSTALL.vcxproj" (1) is building "D:\bugs\windows_example\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +29 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +31 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[   +1 ms] Done Building Project "D:\bugs\windows_example\build\windows\ZERO_CHECK.vcxproj" (default targets).
[ +127 ms] Project "D:\bugs\windows_example\build\windows\INSTALL.vcxproj" (1) is building "D:\bugs\windows_example\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[        ] Project "D:\bugs\windows_example\build\windows\ALL_BUILD.vcxproj" (3) is building "D:\bugs\windows_example\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) on node 1 (default targets).
[        ] Project "D:\bugs\windows_example\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) is building "D:\bugs\windows_example\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +6 ms] CustomBuild:
[        ]   "The build of 'D:\bugs\windows_example\build\windows\CMakeFiles\e443b2b5ccebe6faba37c15bec2afede\flutter_assemble.rule' depends on 'D:\BUGS\WINDOWS_EXAMPLE\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of 'D:\bugs\windows_example\build\windows\CMakeFiles\a2c24e09185a63f4e844e0baf65337a2\flutter_windows.dll.rule'. The items cannot be built in parallel."
[  +71 ms]   Generating D:/bugs/windows_example/windows/flutter/ephemeral/flutter_windows.dll, D:/bugs/windows_example/windows/flutter/ephemeral/flutter_export.h, D:/bugs/windows_example/windows/flutter/ephemeral/flutter_windows.h, D:/bugs/windows_example/windows/flutter/ephemeral/flutter_messenger.h, D:/bugs/windows_example/windows/flutter/ephemeral/flutter_plugin_registrar.h, D:/bugs/windows_example/windows/flutter/ephemeral/flutter_texture_registrar.h, D:/bugs/windows_example/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc, D:/bugs/windows_example/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc, D:/bugs/windows_example/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc, D:/bugs/windows_example/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc, D:/bugs/windows_example/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[+1304 ms]   [ +124 ms] executing: [C:\src\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +104 ms]   [ +133 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ]   [        ] 85684f9300908116a78138ea4c6036c35c9a1236
[        ]   [   +1 ms] executing: [C:\src\flutter/] git tag --points-at 85684f9300908116a78138ea4c6036c35c9a1236
[  +94 ms]   [  +94 ms] Exit code 0 from: git tag --points-at 85684f9300908116a78138ea4c6036c35c9a1236
[        ]   [        ] 3.0.4
[   +9 ms]   [   +9 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +78 ms]   [  +78 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ]   [        ] origin/stable
[        ]   [        ] executing: [C:\src\flutter/] git ls-remote --get-url origin
[  +75 ms]   [  +75 ms] Exit code 0 from: git ls-remote --get-url origin
[        ]   [        ] https://github.com/flutter/flutter.git
[  +65 ms]   [  +65 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref HEAD
[  +73 ms]   [  +73 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ]   [        ] stable
[  +82 ms]   [  +82 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +4 ms]   [   +4 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +117 ms]   [ +117 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +3 ms]   [   +4 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +38 ms]   [  +38 ms] Initializing file store
[  +14 ms]   [  +14 ms] Done initializing file store
[  +63 ms]   [  +63 ms] Skipping target: gen_localizations
[  +12 ms]   [  +12 ms] Skipping target: gen_dart_plugin_registrant
[+2881 ms]   [+2881 ms] Skipping target: unpack_windows
[ +678 ms]   [ +678 ms] Skipping target: kernel_snapshot
[ +323 ms]   [ +323 ms] Skipping target: debug_bundle_windows_assets
[        ]   [        ] Persisting file store
[   +9 ms]   [   +8 ms] Done persisting file store
[   +5 ms]   [   +5 ms] build succeeded.
[   +8 ms]   [   +8 ms] "flutter assemble" took 4,185ms.
[ +143 ms]   [ +143 ms] ensureAnalyticsSent: 139ms
[   +1 ms]   [   +1 ms] Running shutdown hooks
[        ]   [        ] Shutdown hooks complete
[        ]   [        ] exiting with code 0
[  +89 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[        ] Done Building Project "D:\bugs\windows_example\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[   +7 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_app.dir\Debug\flutter_.C3CE7AC5.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +2 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +53 ms] ClCompile:
[        ]   All outputs are up-to-date.
[  +24 ms] Lib:
[        ]   All outputs are up-to-date.
[   +1 ms]   flutter_wrapper_app.vcxproj -> D:\bugs\windows_example\build\windows\flutter\Debug\flutter_wrapper_app.lib
[  +10 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.C3CE7AC5.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_app.dir\Debug\flutter_.C3CE7AC5.tlog\flutter_wrapper_app.lastbuildstate".
[        ] Done Building Project "D:\bugs\windows_example\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[  +43 ms] Project "D:\bugs\windows_example\build\windows\ALL_BUILD.vcxproj" (3) is building "D:\bugs\windows_example\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (6) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.35938FDE.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +3 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +21 ms] ClCompile:
[        ]   All outputs are up-to-date.
[   +5 ms] Lib:
[        ]   All outputs are up-to-date.
[        ]   flutter_wrapper_plugin.vcxproj -> D:\bugs\windows_example\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[   +6 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.35938FDE.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.35938FDE.tlog\flutter_wrapper_plugin.lastbuildstate".
[   +1 ms] Done Building Project "D:\bugs\windows_example\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[  +33 ms] Project "D:\bugs\windows_example\build\windows\ALL_BUILD.vcxproj" (3) is building "D:\bugs\windows_example\build\windows\runner\windows_example.vcxproj" (7) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "windows_example.dir\Debug\windows_example.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +2 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +36 ms] ClCompile:
[        ]   C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\CL.exe /c /ID:\bugs\windows_example\windows /ID:\bugs\windows_example\windows\flutter\ephemeral /ID:\bugs\windows_example\windows\flutter\ephemeral\cpp_client_wrapper\include /Zi /nologo /W4 /WX /diagnostics:column /Od /Ob0 /D WIN32 /D _WINDOWS /D _HAS_EXCEPTIONS=0 /D _DEBUG /D NOMINMAX /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Debug\"" /D _UNICODE /D UNICODE /Gm- /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17 /Fo"windows_example.dir\Debug\\" /Fd"windows_example.dir\Debug\vc142.pdb" /external:W4 /Gd /TP /wd4100 /errorReport:queue D:\bugs\windows_example\windows\flutter\generated_plugin_registrant.cc D:\bugs\windows_example\windows\runner\flutter_window.cpp
[ +107 ms]   generated_plugin_registrant.cc
[ +515 ms]   flutter_window.cpp
[ +859 ms]   Generating Code...
[ +113 ms] ResourceCompile:
[        ]   All outputs are up-to-date.
[  +38 ms] Link:
[        ]   C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE /OUT:"D:\bugs\windows_example\build\windows\runner\Debug\windows_example.exe" /INCREMENTAL /ILK:"windows_example.dir\Debug\windows_example.ilk" /NOLOGO ..\flutter\Debug\flutter_wrapper_app.lib D:\bugs\windows_example\windows\flutter\ephemeral\flutter_windows.dll.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /manifestinput:D:\bugs\windows_example\windows\runner\runner.exe.manifest /DEBUG /PDB:"D:/bugs/windows_example/build/windows/runner/Debug/windows_example.pdb" /SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"D:/bugs/windows_example/build/windows/runner/Debug/windows_example.lib" /MACHINE:X64  /machine:x64 windows_example.dir\Debug\Runner.res
[        ]   windows_example.dir\Debug\flutter_window.obj
[        ]   windows_example.dir\Debug\main.obj
[        ]   windows_example.dir\Debug\utils.obj
[        ]   windows_example.dir\Debug\win32_window.obj
[        ]   windows_example.dir\Debug\generated_plugin_registrant.obj
[ +340 ms]   windows_example.vcxproj -> D:\bugs\windows_example\build\windows\runner\Debug\windows_example.exe
[   +6 ms] FinalizeBuildStatus:
[        ]   Deleting file "windows_example.dir\Debug\windows_example.tlog\unsuccessfulbuild".
[        ]   Touching "windows_example.dir\Debug\windows_example.tlog\windows_example.lastbuildstate".
[   +1 ms] Done Building Project "D:\bugs\windows_example\build\windows\runner\windows_example.vcxproj" (default targets).
[   +8 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +3 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +10 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[   +1 ms] Done Building Project "D:\bugs\windows_example\build\windows\ALL_BUILD.vcxproj" (default targets).
[   +9 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +63 ms] PostBuildEvent:
[        ]   setlocal
[        ]   "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Debug -P cmake_install.cmake
[        ]   if %errorlevel% neq 0 goto :cmEnd
[        ]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[        ]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[  +64 ms]   -- Install configuration: "Debug"
[   +2 ms]   -- Up-to-date: D:/bugs/windows_example/build/windows/runner/Debug/data/icudtl.dat
[        ]   -- Up-to-date: D:/bugs/windows_example/build/windows/runner/Debug/flutter_windows.dll
[   +7 ms]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets
[        ]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[   +3 ms]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[   +2 ms]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/fonts
[        ]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +9 ms]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[ +172 ms]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/NOTICES.Z
[   +3 ms]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/packages
[        ]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[   +1 ms]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets
[   +1 ms]   -- Installing: D:/bugs/windows_example/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[  +19 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[        ] Done Building Project "D:\bugs\windows_example\build\windows\INSTALL.vcxproj" (default targets).
[   +1 ms] Build succeeded.
[        ]     0 Warning(s)
[        ]     0 Error(s)
[        ] Time Elapsed 00:00:09.49
[ +454 ms] Observatory URL on device: http://127.0.0.1:59805/MjmZlL_I_EM=/
[   +5 ms] Caching compiled dill
[  +34 ms] Connecting to service protocol: http://127.0.0.1:59805/MjmZlL_I_EM=/
[ +120 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at http://127.0.0.1:59805/MjmZlL_I_EM=/.
[  +69 ms] DDS is listening at http://127.0.0.1:59809/LYpI6rjNBBE=/.
[  +57 ms] Successfully connected to service protocol: http://127.0.0.1:59805/MjmZlL_I_EM=/
```
</details>

<details>
 <summary>flutter analyze</summary>

```
Analyzing windows_example...
No issues found! (ran in 5.4s)
```
</details>

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 3.0.4, on Microsoft Windows [Version 10.0.22000.739], locale en-150)
    • Flutter version 3.0.4 at C:\src\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 85684f9300 (13 days ago), 2022-06-30 13:22:47 -0700
    • Engine revision 6ba2af10bb
    • Dart version 2.17.5
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\dwyso\AppData\Local\Android\sdk
    • Platform android-31, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[✓] Visual Studio - develop for Windows (Visual Studio Build Tools 2019 16.11.16)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools
    • Visual Studio Build Tools 2019 version 16.11.32602.291
    • Windows 10 SDK version 10.0.19041.0

[✓] Android Studio (version 4.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)

[✓] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22000.739]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 103.0.5060.114
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 102.0.1245.33

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>

**Comment by huycozy** [comment]
Hi @TortillaZHawaii 
I've tried many times but the app didn't freeze. As far as I can see from your video, freeze occurs after an amount of time tries running the application but you didn't snap window at every try.
Can you try again and find clearer steps to reproduce this issue? 
And also retry with the latest stable `3.0.5` and master `3.1.0-0.0.pre.1647` channel to see if the issue still reproduces.
Thanks!

**Comment by TortillaZHawaii** [comment]
Hi @huycozy,
sorry for misleading video, I tried to also show that the issue doesn't occur after moving the apps window by cursor, only after relaunching it after moving it using FancyZones (Windows key + Arrows). So the steps are as specified in the issue, meaning:
> 1. Create an app using `flutter create`.
>  2. Use vscode to launch the app.
>  3. Move launched app using [PowerToys FancyZones](https://docs.microsoft.com/en-us/windows/powertoys/fancyzones).
>  4. Stop the app.
>  5. Launch the app again using vscode.

The issue persist in latest stable channel `3.0.5`, but seems to be gone in master channel, I used `3.1.0-0.0.pre.1651`. 
I also tried recreating it on another machine on stable channel and I was not able to freeze the app.

For now I will use master channel, thanks!
